### PR TITLE
[dhctl] Preflight check exist embedded containerd 

### PR DESCRIPTION
**Pull request by aleksey-su**
## Description
Preflight check exist embedded containerd 

## Why do we need it, and what problem does it solve?
#7960

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: dhctl
type: feature
summary: Preflight check exist embedded containerd 
impact_level: default
```

### CODE REVIEW

**Comment by aleksey-su** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/static'...

**Comment by aleksey-su** [comment]
/e2e/abort static;Static;containerd;1.25 9315937719 9315937719-1-con-1-25 /sys/deckhouse-oss/install:pr8550 astra@192.168.199.115

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/static'...

File: dhctl/pkg/preflight/containerd.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Flant JSC
```
Comment (name212) [inline dhctl/pkg/preflight/containerd.go:1]:
```suggestion
// Copyright 2024 Flant JSC
```

**Review by name212 — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Comment by z9r5** [comment]
/e2e/abort static;Static;containerd;1.25 9316819426 9316819426-2-con-1-25 /sys/deckhouse-oss/install:pr8550 astra@192.168.199.126

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Pull request merged**